styleguide/template.md

```diff
@@ -148,7 +148,10 @@ Examples:
 
 For more information about using this notation, see [Using cross reference](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#using-cross-reference).
 
-> Right now, there is no easy way to find the UIDs. The best way to find the UID for an API is to view the source for the API page you want to link to and find the ms.assetid value. Individual overload values are not shown in the source. We're working on having a better system in the future.
+There are two ways to find the UID:
+
+- View the source for the API page you want to link to and find the ms.assetid value. Note that individual overload values are not shown in the source.
+- Use the following tool to search for UIDs: https://xref.docs.microsoft.com/autocomplete?text=tostring (replace tostring with parts of the API name you're trying to find).
```

Comment:
My second comment seems to have gotten lost. But I thought it might be helpful to provide two replacement examples to make it clear that the replacement string can be not only a member name (like TryParse) but also a type and member name (like Int32.TryParse).